WeScan/ImageScannerController.swift

```diff
@@ -85,6 +92,8 @@ public final class ImageScannerController: UINavigationController {
     }
 
     override public init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        self.imageScannerOptions = ImageScannerOptions()
+        
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
     }
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: WeScan/ImageScannerController.swift;
  Line: 99;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)